alarms.json.default

```diff
@@ -207,5 +207,158 @@
 		"Dragonite":"False",
 		"Mewtwo":"False",
 		"Mew":"False"
+	},
+	"ivs":{
```

Comment:
Instead of adding a different section, I think it would be better to change each Pokemon into an object to filter based on that. For example, it could look like the following:
```json
"Bulbasaur":{
    "dist":"0",
    "iv":"90.0",
    "move_1": ["Tackle", "Vine Whip"],
    "move_2" : ["Seed Bomb", "Sludge Bomb"]
}
```